CODEOWNERS

```diff
@@ -26,3 +26,6 @@ zh-cn/ @stevenlele
 
 # Slovak
 sk/ @kubo6472
+
+# Toki Pona
+tp/ @woflydev
```

Comment:
the iso code for toki pona is `tok`, not `tp`

Reply:
how the hell did you manage to pop up in my github emails again jeezus

i didnt bother looking up the code, tp was the first thing that came to mind.